src/cfml/system/services/ServerService.cfc

```diff
@@ -703,6 +703,21 @@ component accessors="true" singleton {
 		serverJSON.trayOptions = serverJSON.trayOptions.map( function( item ){
 			if( item.keyExists( 'image' ) && item.image.len() ) {
 				item.image = fileSystemUtil.resolvePath( item.image, defaultServerConfigFileDirectory );
+			}else{
+				if( item.keyExists( 'action' )){
+					switch(item.action){
+						case 'run':
+						case 'runAsync':
+						case 'runTerminal':
```

Comment:
We still need the runTerminal implementation removed from runwar and added to CommandBox as was the plan all along.  This includes new top-level CommandBox settings to control the script for opening a native terminal.